PosteAPI/views.py

```diff
@@ -334,6 +339,67 @@ def post(self, request):
             # error message contained in response.data
             return response
 
+    def patch(self, request):
+        """
+        Edits a post in the server
+        :param request: Request object with post id and auth in header
+                        Request Body contains new title, description and url
+        """
+        #print(request)
+        try:
```

Comment:
As a general suggestion (non-blocking), try to keep `try` blocks as short as possible. At my workplace, they limit us to ONE line in a try block -- any longer and we need to either break things out of the try block (perhaps into separate try blocks of their own) or create a function. This helps isolate what code breaks something / needs to catch an exception // abstracts error handling. I don't consider it a blocker here, but think it might be a good learning exercise. I'm still guilty myself about not following this convention closely enough, but they make good points for it...

Reply:
I also try to keep it as short as possible. 
I thought 5 LOC was short enough, but I guess we can always go shorter